### PR TITLE
Ginkgo: check helpers initialization are correct

### LINKED ISSUE #2065: Ginkgo CI: tests need to check if cilium and docker helpers are nil
The cilium / docker helpers for each Ginkgo tests can return be nil, but the tests don't check this. Add checks for if the helpers are nil, e.g.:
```
	initialize := func() {
		if initialized == true {
			return
		}
		logger = log.WithFields(log.Fields{"test": "RuntimeLB"})
		logger.Info("Starting")
		docker, cilium = helpers.CreateNewRuntimeHelper(helpers.Runtime, logger)
		cilium.WaitUntilReady(100)
		docker.NetworkCreate(helpers.CiliumDockerNetwork, "")
		initialized = true
	}
```

The tests should exit immediately (maybe `os.Exit(-1)`) as this means the VM configuration is incorrect. Without error checking, the tests continue to run and panic, which is confusing and hard to debug.

### PR DESCRIPTION
**Pull request by eloycoto**
Added a check on `CreateKubectl` and `CreateNewRuntimeHelper` to make
sure that the ssh meta information can be retrieved correctly and
commands can be executed correctly.

Fix #2065

### CODE REVIEW

File: test/helpers/kubectl.go [hunk 1]
```diff
@@ -49,9 +50,18 @@ type Kubectl struct {
...
 // CreateKubectl initializes a Kubectl helper with the provided vmName and log
+// It marks the test as Fail if cannot get the ssh meta information or cannot
+// execute a `ls` on the virtual machine.
 func CreateKubectl(vmName string, log *logrus.Entry) *Kubectl {
 	node := GetVagrantSSHMeta(vmName)
 	if node == nil {
+		ginkgo.Fail(fmt.Sprintf("Cannot connect to vmName  '%s'", vmName), 1)
+		return nil
+	}
+	res := node.Exec("ls /tmp/")
```
Comment (ianvernon) [inline test/helpers/kubectl.go]: What is the reason for checking the result of this command? It seems arbitrary

Comment (eloycoto) [reply test/helpers/kubectl.go]: Hi @ianvernon 

I saw a few times that the ssh meta information was correct but can't execute commands, and all commands start to fail. 

That `ls` make sure that can ssh into and run commands without problems, and prevent further issues.

Comment (ianvernon) [reply test/helpers/kubectl.go]: OK, so this is a basic sanity check. Can you add a comment equivalent to your above explanation to each place where you do this `ls` check?

**Comment by eloycoto** [comment]
@ianvernon should be ready to merge ;-)

**Review by ianvernon — APPROVED** [review]

**Pull request merged**